pandas/tests/io/pytables/store/test_append.py

```diff
@@ -0,0 +1,931 @@
+import datetime
+from datetime import timedelta
+import re
+from warnings import catch_warnings
+
+import numpy as np
+import pytest
+
+from pandas._libs.tslibs import Timestamp
+
+import pandas as pd
+from pandas import (
+    DataFrame,
+    MultiIndex,
+    Series,
+    _testing as tm,
+    concat,
+    date_range,
+    read_hdf,
+)
+from pandas.tests.io.pytables.common import (
+    _maybe_remove,
+    ensure_clean_path,
+    ensure_clean_store,
+)
+
+
+@pytest.mark.single
+class TestAppend:
```

Comment:
not really a fan of the classes, ideally just make the test structure flat. you can do the mark on the entire file.

Reply:
same for all below

Reply:
flattened the test structure in all files - now all tests that were in test_store are functions, not methods of classes